### DISCUSSION TITLE
Any way to chat with the tactical rmm community without using Discord?

### DISCUSSION THREAD

**Discussion by zaggynl** [amidaware/tacticalrmm#1054] [Ideas]
I'm not using Discord, any way to chat via something like an IRC or Matrix bridge?

**Comment by dinger1986** [comment]
Discord only at the moment.

**Comment by johnnyq** [comment]
A forum would be awesome!

**Comment by bbrendon** [comment]
A semi-forum . https://github.com/amidaware/tacticalrmm/discussions

**Comment by dinger1986** [comment]
> A semi-forum . https://github.com/amidaware/tacticalrmm/discussions

It's not really the community though, it's us guys who do support. 

For "chatting" to the community it's discord.

Forum etc will come in the fullness of time but at the moment discord is enough to handle at times 😃.
